src/components/Header.js

```diff
@@ -1,6 +1,11 @@
+// @flow
 import React from "react";
 
-const Header = props => (
+type Props = {
+  ImageUrl: string
+};
+
+const Header = (props: Props) => (
```

Comment:
please always deconstruct props:
`({ imageUrl }: Props)`

also note that props should be always lowercase (`imageUrl`). please fix both things everywhere

Reply:
1a1959804e2116399474c6eefca0190c38da9e23
73c0ccf7a30dadf6ad725c5c099e4cb312110cf9
5cd2c65184addc5b809ebf785a3b091c20acd6ad